utils/kafka-test-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala

```diff
@@ -51,24 +52,20 @@ class KafkaClient(kafkaAddress: String, id: String) extends LazyLogging {
     promise.future
   }
 
-  def sendMessage[T: Encoder](topic: String, content: T): Future[RecordMetadata] =
+  def sendMessage[T: Encoder: ClassTag](topic: String, content: T): Future[RecordMetadata] =
```

Comment:
Why do you need to add a ClassTag?

Reply:
Because right now to encode to string (beloved) we use ConsumerRecordHelper.asString

Reply:
Hmm, you're right -> we don't need it.